benchmark/sampler/neighbor.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import time
+
+import torch
+# uncomment below to enable torch.ops.torch_sparse.neighbor_sample
+# import torch_sparse
+from tqdm import tqdm
+
+import pyg_lib
+from pyg_lib.testing import withDataset, withSeed
+
+argparser = argparse.ArgumentParser('Neighbor Sampler benchmark')
+argparser.add_argument('--batch-sizes', nargs='+',
+                       default=[512, 1024, 2048, 4096, 8192], type=int)
+argparser.add_argument('--num_neighbors', default=[[-1], [15, 10, 5],
+                                                   [20, 15, 10]], type=int)
+argparser.add_argument('--replace', default=False, type=bool)
+argparser.add_argument('--directed', default=True, type=bool)
+
+args = argparser.parse_args()
+
+
+@withSeed
+@withDataset('DIMACS10', 'citationCiteseer', True)
```

Comment:
```suggestion
@withDataset('DIMACS10', 'citationCiteseer', return_csc=True)
```
nit, since I was confused by L26